[Demo] Add last changes to visual-vocabulary before publishing (#693)
mckinsey/vizro: vizro-core/README.md

<br/>

## Vizro examples gallery

You can see Vizro in action by clicking on the following image or by visiting [the examples gallery at vizro.mckinsey.com](https://vizro.mckinsey.com).

<p align="center">
<a href="http://vizro.mckinsey.com/">
<img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/vizro_examples_gallery.png" width="900">
</a>
</p>

## Vizro visual vocabulary

> Our visual vocabulary dashboard is a guide for selecting and creating various types of charts. It provides guidance

Reviewer: ```suggestion
Our visual vocabulary dashboard helps you to select and create various types of charts. It provides guidance
```
